src/Propulsion/Infrastructure.fs

```diff
@@ -1,4 +1,14 @@
+#if CORE
 namespace Propulsion
```

Comment:
Ah, I see what you're trying to do now 
remember, this is all `internal` - if you call `Propulsion.Core`, it wont offend anyone - having changes in the ES version for is perfectly valid